h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -238,6 +242,11 @@ public void compute2() {
         Scope.exit();
       }
       tryComplete();
+      if(callback != null) {
+        final ModelBuildingResult modelBuildingResult = new ModelBuildingResult(Optional.of(_job.get()),
```

Comment:
When can job for a model be null?

Reply:
This branch is not exceptional completion, which means the model will be there. The Optional is there for the consumer to honor the contract - The ModelBuildingResult might as well contain an exception.